tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/LiferayWorkspaceUtil.java

```diff
@@ -132,7 +132,7 @@ public static String getGradleProperty(String projectLocation, String key, Strin
 		return retVal;
 	}
 
-	public static String getHomeDir(String location) {
+	public static String getBundlesDir(String location) {
```

Comment:
We shouldn't hardcode this as bundles.  This location can be configured.  We should look it up via the gradle.properties.  cc @jtydhr88

Reply:
Please resend, thanks!

Reply:
hey greg, we has already get it from  gradle.properties.